### ISSUE TITLE
Duplicate feedstock?

### ISSUE THREAD

**Issue by m-rossi** [conda-forge/lsd-rust-feedstock#3]
In a [recent zulip discussion](https://conda-forge.zulipchat.com/#narrow/stream/457337-general/topic/using.20cargo.2Etoml) I discovered this feedstock, which seems to be a duplicate of the feedstock I created a while ago: https://github.com/conda-forge/lsdeluxe-feedstock.

It is a little confusing as the source repository changed its name and organization.

@danielnachun may we should request archiving the feedstock, update the url and add you as a maintainer in _my_ feedstock?

**Comment by danielnachun** [comment]
Yes, unfortunately I totally missed that the tool changed its name.  Confusingly, https://github.com/conda-forge/lsdeluxe-feedstock also a has a different upstream URL, but that URL just redirects to the on used here.  So both the package name and upstream URL have changed, leading to the confusion.

One thing that would be nice to figure out is if there is a way for `lsdeluxe` to also have an alias for `lsd-rust` so that users understand it is the same tool.  Nevertheless this feedstock is redundant and should be archived.

**Comment by m-rossi** [comment]
> Yes, unfortunately I totally missed that the tool changed its name.  Confusingly, https://github.com/conda-forge/lsdeluxe-feedstock also a has a different upstream URL, but that URL just redirects to the on used here.  So both the package name and upstream URL have changed, leading to the confusion.

That's intentional behavior to minimize errors due to transfer of repositories to another user/organization.

> One thing that would be nice to figure out is if there is a way for `lsdeluxe` to also have an alias for `lsd-rust` so that users understand it is the same tool.  Nevertheless this feedstock is redundant and should be archived.

That's possible with [multi-output-recipes](https://conda-forge.org/docs/maintainer/knowledge_base/#multi-output-recipes).

I chose `lsdeluxe` in the PR, due to name conflicts: https://github.com/conda-forge/staged-recipes/pull/18107.

I would be okay with `lsd-rs` as additional package output as it's also [the name of the organization](https://github.com/lsd-rs).

Furthermore we should also add a PR in the source repo once we are done her and add install instructions for conda-forge the prohibit future duplicates.

**Comment by danielnachun** [comment]
That all sounds good to me!  I've opened https://github.com/conda-forge/admin-requests/pull/1189 to archive this feedstock.
